UPGRADE-4.0.md

```diff
@@ -297,26 +297,24 @@ Form
    `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed.
 
  * Using callable strings as choice options in ChoiceType is not supported
-   anymore in favor of passing PropertyPath instances.
+   anymore in favor of passing \Callable.
```

Comment:
this `\Callable` does not exist and it might confusing ppl, what about remove it? `[...] anymore.`